source/blog/2017-06-06-using-kubeadm-with-cri-o.html.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Using kubeadm with CRI-O
+author: runcom
+date: 2017-06-06 12:58:13 UTC
+comments: true
+published: true
+---
+
+**[CRI-O](https://github.com/kubernetes-incubator/cri-o)** is a Kubernetes incubator project which is meant to provide an integration path between Open Containers Initiative (OCI) conformant runtimes and the kubelet. Specifically, it implements the Container Runtime Interface (CRI) using OCI conformant runtimes. CRI-O uses [runc](https://github.com/opencontainers/runc) as its default runtime to run Kubernetes pods. For more information you can read a brief introduction [here](https://www.projectatomic.io/blog/2017/02/crio-runtimes/). If you're interested into why you should use CRI-O instead of other container runtimes you can read more [here](https://www.projectatomic.io/blog/2017/06/6-reasons-why-cri-o-is-the-best-runtime-for-kubernetes/).
+
+Introduction
+=
+
+First things first, I'm pleased to announce that [CRI-O](https://github.com/kubernetes-incubator/cri-o) can be seamlessy used with [kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/) to install kubernetes. This allows containers to be run under Kubernetes without the docker-engine being present on your servers.
+
+
+Getting  started with CRI-O and `kubeadm`
+=
+
+To begin, you first provision machines with CRI-O using a [handy ansible playbook](https://github.com/cri-o/cri-o-ansible). The playbook will compile CRI-O from source and is the only installation method currently available. Installation packages for all of the major distributions are under development and should be available in the near future. Stay tuned!
+
+Once you have  successfully installed CRI-O, you should  follow the `kubeadm` [official documentation](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/) to install kubernetes. You must also add a [network add-on](https://kubernetes.io/docs/concepts/cluster-administration/addons/) to provide networking for the cluster.
```

Comment:
Aha, I see, you do provide alternate instuctions below.  It would be better just to reference these alternate instructions and not link the official kubeadm docks at all, especially since following them will get users into trouble.